dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -176,6 +177,19 @@ interface RumMonitor {
      */
     class Builder {
 
+        private var samplingRate: Float = RumFeature.samplingRate
+
+        /**
+         * Sets the sampling rate for RUM Sessions.
+         *
+         * @param samplingRate the sampling rate must be a value between 0 and 100. A value of 0
+         * means no RUM event will be sent, 100 means all sessions will be kept.
+         */
+        fun sampleRumSessions(@FloatRange(from = 0.0, to = 100.0) samplingRate: Float): Builder {
```

Comment:
Then we should remove the related code in the `DatadogConfig` and `RumFeature` classes